TypeScript errors in turf 6 (TS2304 and TS2312)

I've updated @turf/turf from 5.1.6 to 6.3.0 and now TypeScript is complaining about the following when I transpile my project:

```
node_modules/@turf/center-median/dist/js/index.d.ts:59:29 - error TS2304: Cannot find name 'Position'.

59     medianCandidates: Array<Position>;
                               ~~~~~~~~

node_modules/@turf/clusters-dbscan/dist/js/index.d.ts:3:38 - error TS2312: An interface can only extend an object type or intersection of object types with statically known members.

3 export interface DbscanProps extends Properties {
                                       ~~~~~~~~~~

node_modules/@turf/clusters-kmeans/dist/js/index.d.ts:2:38 - error TS2312: An interface can only extend an object type or intersection of object types with statically known members.

2 export interface KmeansProps extends Properties {
                                       ~~~~~~~~~~
```

My tsconfig.json:
```
{
  "compilerOptions": {
    "module": "commonjs",
    "esModuleInterop": true,
    "allowJs": false,
    "allowSyntheticDefaultImports": true,
    "target": "es2019",
    "noImplicitAny": true,
    "strictNullChecks": true,
    "noImplicitThis": true,
    "noImplicitReturns": true,
    "moduleResolution": "node",
    "preserveSymlinks": true,
    "sourceMap": true,
    "outDir": "dist",
    "baseUrl": ".",
    "paths": {
      "*": [
        "node_modules/*",
        "src/types/*",
      ]
    },
    "typeRoots": [
      "./node_modules/@types",
      "./src/types"
    ]
  },
  "include": [
    "src/**/*",
    "test/**/*"
  ]
}
```

I just tried with 6.2.0 and that gives the same issue. For now, I've reverted back to 5.1.6

--- a/packages/turf-center-median/index.ts
+++ b/packages/turf-center-median/index.ts
@@ -9,6 +9,7 @@ import {
   FeatureCollection,
   Feature,
   Point,
+  Position,
 } from "@turf/helpers";
 import { featureEach } from "@turf/meta";
 

